src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -12280,15 +12280,15 @@ void writeVeriSumSpaceTables(EnergyPlusData &state, bool produceTabular, bool pr
         state.dataOutRptTab->zstArea(iTotal) = 0.0;
         state.dataOutRptTab->zstLight(iTotal) = 0.0;
         state.dataOutRptTab->zstPlug(iTotal) = 0.0;
-        state.dataOutRptTab->zstArea(iTotal) = 0.0;
+        state.dataOutRptTab->zstPeople(iTotal) = 0.0;
     }
```

Comment:
This is where the bug is actually located. The correct initialization variable should be `state.dataOutRptTab->zstPeople` instead of `state.dataOutRptTab->zstArea`. It is obvious that the original code got a typo that cause the problem.

With the bug, an important factor `state.dataOutRptTab->zstPeople` would not be cleared out properly and the people in the space would be double counted. The resulting people density is then reported twice as much as the correct number.